agents/mongodb/internal/profiler/aggregator/aggregator.go

```diff
@@ -251,13 +252,15 @@ func (a *Aggregator) createResult(ctx context.Context) *report.Result {
 		s := strings.SplitN(v.Namespace, ".", 2)
 		if len(s) == 2 {
 			db = s[0]
-			collection = s[1]
+			collection, _ = truncate.Query(s[1])
 		}
 
+		example, _ := truncate.Query(v.Query)
+		fingerprint, _ := truncate.Query(v.Fingerprint)
 		bucket := &agentpb.MetricsBucket{
 			Common: &agentpb.MetricsBucket_Common{
```

Comment:
we have isTruncated here and it should be used here.

Reply:
if fingerprint or example is truncated `isTruncated` should be true.

Reply:
Here and in other agents

Reply:
Why did you resolve this? I see that we still don't use isTruncated

Reply:
I resolve it yesterday after I pushed commit "PMM-5365 Patch." because truncating was not in this commit.

Reply:
@BupycHuk Just some clarification. If example return truncated true and fingerprint false, what is right behavior? If any will be true then I always return true?

Reply:
yes, if any of them is true then it should be true

Reply:
well, that wasn't correct :)

Reply:
@BupycHuk I think we should revert fingerprint truncation – and maybe even query truncation as it is should not be possible to have UTF-8 related problems in MongoDB – it uses _only_ UTF-8.